pynitrokey/cli/update.py

```diff
@@ -61,17 +65,26 @@ def update(serial, yes):
         client = find(serial)
 
     except pynitrokey.exceptions.NoSoloFoundError as e:
-        local_critical(None,
-            "No Nitrokey key found!", e, None,
+        local_critical(
+            None,
+            "No Nitrokey key found!",
+            e,
+            None,
```

Comment:
I wanted to complain about formatting, but I guess this calls for the `local_critical` refactoring.